### PR TITLE
fix(DIST-1643): Change default sidetab icon size

### PR DESCRIPTION
**Pull request by mathio**
Before:
Screenshot 2022-07-19 at 11 23 12

After:
Screenshot 2022-07-19 at 11 34 29

### CODE REVIEW

**Review by scarciofolomarco — APPROVED** [review]

File: packages/embed/e2e/applitools-utils.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { screenSizeDesktop, screenSizeMobile, Viewport } from './cypress-utils'
 
 const eyesCheck = async (name: string, { width, height }: Viewport) => {
+  cy.wait(2000)
```
Comment (antoniofull) [inline packages/embed/e2e/applitools-utils.ts:4]: wouldn't be better to wait for the element? I believe Cypress has a 5 seconds default time

**Review by antoniofull — APPROVED** [review]

File: packages/embed/e2e/applitools-utils.ts (hunk 1 above)
Comment (mathio) [reply packages/embed/e2e/applitools-utils.ts:4]: I tried, but It looks like the wait is best solution.

Comment (antoniofull) [reply packages/embed/e2e/applitools-utils.ts:4]: 👍

**Pull request merged**